packages/common/src/directives.ts

```diff
@@ -41,5 +41,6 @@ export const schemaPrepend = gql`
     visualizations: Boolean
     icon: String
     autoSync: Boolean
+    label: String
```

Comment:
Can you include this in the tests?
You can simply update 

```
packages/plugin-terraform/test/testdata/base.graphql
``` 

and 

```
packages/plugin-terraform/test/testdata/expected/base.tf
```

and run `pnpm test` in the terraform folder to verify

Reply:
Furthermore, can you run `pnpm changeset`?
This will generate a changeset file that you have to include in this PR please.